resource/static/vocabulary/timescale/isc2018-1.ttl

```diff
@@ -5809,8 +5809,7 @@ isc:Capitanian
   skos:prefLabel "卡匹敦期"@zh ;
   time:hasBeginning isc:BaseCapitanian ;
   time:hasEnd isc:BaseLopingian ;
-  time:intervalDuring isc:Guadalupian ;
-  time:intervalIn isc:Guadalupian ;
+  time:intervalFinishes isc:Guadalupian ;

```

Comment:
OK. Good catch

Reply:
The `time:intervalIn` axiom is not incorrect, but is redundant with the `time:intervalFinishes` axiom

Reply:
> The `time:intervalIn` axiom is not incorrect, but is redundant with the `time:intervalFinishes` axiom

That is why I suggested it’s removal. Don’t you agree it should be removed?

Reply:
There is no need to remove it. It does not introduce any inconsistencies.